### DISCUSSION TITLE
Daily 7/12/2023

### DISCUSSION THREAD

**Discussion by jovanHillion** [Sinan-Karakaya/R-Type#51] [Daily]
Hello cette aprèm j'avance sur le test du scripting

**Comment by lucasdpt** [comment]
Suite de la lib réseau, je commence la gestion des Packets

**Comment by Sinan-Karakaya** [comment]
Part time, mais je continue de bosser quand je peux

- Une PR que j'ai ouverte pour ImGui-SFML est en train d'être accepté, j'attends juste la réponse des maintainers (elle était nécessaire pour résoudre un bug dans l'assetExplorer de l'éditeur
- Ajout des milestones de sprint et de + de labels sur le repo
- On va continuer à bosser sur l'éditeur ce soir si j'ai le temps et la force
